### PR TITLE
Add email article module moving AB test functionality perm

### PR DESCRIPTION
**Pull request by gtrufitt**
This is moving towards removing the email in-article AB test, making the functionality perm, initially by making the NHS email list appear in NHS related articles. When we have a conclusion to the AB test, we'll roll required functionality into this module.
- [x] wrap in switch
- [x] fill in NHS list details
- [x] double-check failsafes

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/email/email-article.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            return $el.nodeName && $el.nodeName === 'P';
+        },
+        listCanRun = function (listConfig) {
+            if (listConfig.canRun && canRunList[listConfig.canRun]()) {
+                return listConfig;
+            }
+        },
+        addListToPage = function (listConfig) {
+            if (listConfig) {
+                var iframe = bonzo.create('<iframe src="/email/form/article/' + listConfig.listId + '" height="218px" data-form-title="' + listConfig.headline + '" data-form-description="' + listConfig.description + '" data-form-campaign-code="' + listConfig.campaignCode + '" scrolling="no" seamless frameborder="0" class="iframed--overflow-hidden email-sub__iframe js-email-sub__iframe js-email-sub__iframe--article"' + ' data-form-success-headline="' + listConfig.successHeadline + '" data-form-success-desc="' + listConfig.successDescription + '"></iframe>')[0];
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/email/email-article.js]: wanna use a template for this?

Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/email/email-article.js]: Yes. Yes I do.

File: static/src/javascripts/projects/common/modules/email/email-article.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+                bean.on(iframe, 'load', function () {
+                    email.init(iframe);
+                });
+
+                fastdom.write(function () {
+                    $(iframe).appendTo($articleBody);
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/email/email-article.js]: maybe do the dom lookup outside the callback, to minimise the activity tied to the frame?

**Comment by sndrs** [comment]
:+1: just suggestions, not biggies

**Pull request merged**